app/models/user.rb

```diff
@@ -7,6 +7,7 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable, :jwt_authenticatable, jwt_revocation_strategy: Blacklist
   belongs_to :company
   belongs_to :user_role
+  belongs_to :warehouse
```

Comment:
Is this really the right action? It deprives us of the ability to create a user without a warehouse.